awx/api/serializers.py

```diff
@@ -4687,13 +4687,15 @@ def validate(self, attrs):
         return attrs
 
     def create(self, validated_data):
+        request = self.context.get('request', None)
+        launch_user = request.user if request else None
         job_node_data = validated_data.pop('jobs')
         wfj_deferred_attr_names = ('skip_tags', 'limit', 'job_tags')
         wfj_deferred_vals = {}
         for item in wfj_deferred_attr_names:
             wfj_deferred_vals[item] = validated_data.pop(item, None)
 
-        wfj = WorkflowJob.objects.create(**validated_data, is_bulk_job=True)
+        wfj = WorkflowJob.objects.create(**validated_data, is_bulk_job=True, launch_type='manual', created_by=launch_user)
```

Comment:
this fixes it so in the tests we always get the launch type and created_by set so later the non admin or org admin users can see result